slices/libglib2.0-0t64.yaml

```diff
@@ -0,0 +1,29 @@
+package: libglib2.0-0t64
+
+essential:
+  - libglib2.0-0t64_copyright
+
+slices:
+  core:
```

Comment:
interesting - first use of a core slice for a libs package :)

Reply:
I had to split this one as core is needed for java.base, but the rest is only relevant for awt =)